CMakeLists.txt

```diff
@@ -17,19 +17,17 @@ project(JPScore LANGUAGES CXX)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
+set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake_modules")
 
+include(helper_functions)
 ################################################################################
 # Optional features
 ################################################################################
 set(JPSFIRE OFF CACHE BOOL "Build with jps fire support")
-message(STATUS "JPSFIRE=${JPSFIRE}")
+print_var(JPSFIRE)
 
-# Options for  jpscore
-if(NOT AIROUTER)
-    set(AIROUTER false)
-else()
-    set(AIROUTER true)
-endif()
+set(AIROUTER OFF CACHE BOOL "Build with AI router support")
+print_var(AIROUTER)
 
 message(STATUS "BUILD_TESTING01: " ${BUILD_TESTING})
```

Comment:
`print_var(BUILD_TESTING)`?

Reply:
you can find the function in `cmake_modules/helper_functions.cmake`
It prints the variable in the form `VAR=123`

Reply:
Nice function! I mean to replace that `message`-statement with
﻿`﻿print_var(BUILD_TESTING)`

Reply:
Oh i misread that, makes much more sense now. I think you didn't look at the latest commit, In the meantime I did remove all `BUILD_TESTING` messages, only one in the form of `print_var` is remaining.